.hooks-config

```diff
@@ -1,5 +0,0 @@
-[hooks "chain"]
```

Comment:
Is there a reply missing here? I just see @aashish24 asking a question and then replying "I see" -- what happened in between?